src/lib/yacc/grammar.rs

```diff
@@ -1084,18 +1086,6 @@ mod test {
         assert_eq!(scores[usize::from(grm.nonterm_idx(&"E").unwrap())], 1);
     }
 
-    #[test]
```

Comment:
Why did you delete this test instead of just changing u64 to u32?

Reply:
The term costs are now `u8`: we'd need to have a grammar rule with over 16 million of them to trigger overflow. That seemed like too big for a test to bother worrying about ;)

Reply:
Ah, didn't see the `should_panic`. That's ok then.